### PR TITLE
React router

### PR DESCRIPTION
**Pull request by Hecktors**

### CODE REVIEW

File: src/pages/Session.js [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+  function getRandomMove() {
+    const movesNum = selectedMoves.length
+    const randomNum = Math.floor(Math.random() * movesNum)
+    return selectedMoves[randomNum]
+  }
+
+  function startMoveTimeout(ms) {
+    timeoutRef.current = setTimeout(() => {
+      const nextCurrentMove = getRandomMove()
+      console.log(nextCurrentMove)
```
Comment (CinFrei) [inline src/pages/Session.js:63]: Found a console.log(counter=1).

File: src/App.js [hunk 2]
```diff
@@ -1,113 +1,55 @@
...
-      startMoveTimeout(nextCurrentMove.steps * 1000)
-    }, ms)
-  }
-
-  function stopMoveAudioProcess() {
-    clearTimeout(timeoutRef.current)
-    timeoutRef.current = null
-    moveAudioRef.current = null
+    setSelectedMoves(getLocalStorage('selectedMoves') ?? [])
+  }, []) // eslint-disable-line react-hooks/exhaustive-deps
```
Comment (CinFrei) [inline src/App.js:15]: Nice hack!

File: src/assets/img/arrow_down.svg [hunk 3]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" fill="white" width="18px" height="18px"><path d="M0 0h24v24H0V0z" fill="none"/><path d="M7.41 8.59L12 13.17l4.59-4.58L18 10l-6 6-6-6 1.41-1.41z"/></svg>
```
Comment (CinFrei) [inline src/assets/img/arrow_down.svg:1]: Here is a little hack to make the svgs smaller. [Have fun!](https://www.svgminify.com/)

File: src/components/Accordion.js [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+      <ul>{listItems}</ul>
+    </LevelAccordionStyled>
+  )
+}
+
+const LevelAccordionStyled = styled.div`
+  padding: 10px;
+  border-radius: 5px;
+  margin-bottom: 10px;
+  background-color: var(--bg-color-accordion);
```
Comment (CinFrei) [inline src/components/Accordion.js:65]: Color var's FTW! Yay :D

File: src/components/Accordion.js [hunk 5]
```diff
@@ -0,0 +1,112 @@
...
+    }
+
+    svg {
+      transform: scale(2);
+    }
+  }
+
+  ul {
+    overflow: hidden;
+    height: ${(props) => (props.isOpen ? 'auto' : 0)};
```
Comment (CinFrei) [inline src/components/Accordion.js:92]: Nice, your a hacker. :rocket:

File: src/components/Header.js [hunk 6]
```diff
@@ -1,29 +1,28 @@
 import styled from 'styled-components/macro'
-import HeaderButton from './HeaderButton'
 import PropTypes from 'prop-types'
 
-Header.propTypes = {
-  title: PropTypes.string.isRequired,
-  isPaused: PropTypes.bool.isRequired,
-  handleClick: PropTypes.func.isRequired,
-}
+Header.propTypes = {}
```
Comment (CinFrei) [inline src/components/Header.js:4]: No PropTypes set, that's sad. :(

File: src/components/__snapshots__/Accordion.test.js.snap [hunk 7]
```diff
@@ -0,0 +1,44 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Accordion renders Accordion 1`] = `
+<div
```
Comment (CinFrei) [inline src/components/__snapshots__/Accordion.test.js.snap:4]: See no styles here. :thinking_face: 
Maybe you could add `/** @component */` to your `Accordion.js`. :)

File: src/lib/getLocalStorage.js [hunk 8]
```diff
@@ -0,0 +1,8 @@
+export default function getLocalStorage(key) {
+  const data = localStorage.getItem(key)
+  try {
+    return JSON.parse(data)
+  } catch (error) {
+    console.error(error)
+  }
```
Comment (CinFrei) [inline src/lib/getLocalStorage.js:7]: Gotta catch 'em all. :charamander_dance:

**Review by CinFrei — COMMENTED** [review]
Great Work 🚀

File: src/components/Layout.js [hunk 9]
```diff
@@ -0,0 +1,25 @@
...
+    position: relative;
+    overflow: auto;
+  }
+
+  footer {
+    width: 100%;
+    display: grid;
+    place-items: center;
+  }
+`
```
Comment (omorgenroth) [inline src/components/Layout.js:25]: You could maybe think about renaming it to "LayoutContainer" (or sth. similar). Would sound a bit more like a reusable component.

File: src/App.js [hunk 10]
```diff
@@ -1,113 +1,55 @@
...
-    </Container>
+      />
+      <Route
+        exact
+        path="/session"
+        render={(props) => <Session {...props} selectedMoves={selectedMoves} />}
+      />
+    </Switch>
   )
 }
```
Comment (omorgenroth) [inline src/App.js:55]: Nice job in shrinking down the amount of Code in the App.js. Looks a lot cleaner and tidier!

File: src/pages/Home.js [hunk 11]
```diff
@@ -0,0 +1,46 @@
...
+    ) : (
+      <div className="msg success">Ready? Click Play to start!</div>
+    )
+
+  return (
+    <Layout>
+      <Header title="Salsa time">
+        <div />
+        <h1 className="logo">Salsa time!</h1>
+        <Button data-testid="btn-settings" onClick={openSettings} isSmall>
```
Comment (omorgenroth) [inline src/pages/Home.js:31]: If you use this kind of Button more often, you could think about just using a <Link to="/settings" /> (and style it the same) to save that extra function.

**Review by omorgenroth — APPROVED** [review]
Looks pretty good overall, saw some stuff i could use myself :)

File: src/assets/img/arrow_down.svg (hunk 3 above)
Comment (Hecktors) [reply src/assets/img/arrow_down.svg:1]: Thanks!

File: src/components/Header.js (hunk 6 above)
Comment (Hecktors) [reply src/components/Header.js:4]: Really sad. Going to change it!

File: src/pages/Session.js (hunk 1 above)
Comment (Hecktors) [reply src/pages/Session.js:63]: Already removed it all!

File: src/lib/getLocalStorage.js (hunk 8 above)
Comment (Hecktors) [reply src/lib/getLocalStorage.js:7]: Oh dear...

**Pull request merged**